notify/wechat/wechat.go

```diff
@@ -94,9 +97,8 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 	if err != nil {
 		return false, err
 	}
-
-	// Refresh AccessToken over 2 hours
-	if n.accessToken == "" || time.Since(n.accessTokenAt) > 2*time.Hour {
+	// Refresh AccessToken over `expireIn`
+	if n.accessToken == "" || time.Since(n.accessTokenAt) > n.expireIn {
```

Comment:
It may be safer to renew the token before it expires (for instance when it reached 90% of its validity).